keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/api/ec2/ClusterSpecTests.kt

```diff
@@ -183,29 +112,75 @@ private data class ResolvedSpecFixture(
   val usEastImageId = "ami-6874986"
   val usWestImageId = "ami-6271051"
 
-  val cloudDriverService = mockk<CloudDriverService>()
-  private val cloudDriverCache = MemoryCloudDriverCache(cloudDriverService)
-
-  fun resolve(): Set<ServerGroup> = spec.resolve(
-    ResolvedImages(
-      appVersion,
-      mapOf(
-        "us-east-1" to usEastImageId,
-        "us-west-2" to usWestImageId
+  val spec: ClusterSpec = ClusterSpec(
+    moniker = Moniker(
+      app = "fnord",
+      stack = "test"
+    ),
+    imageProvider = ArtifactImageProvider(DeliveryArtifact("fnord", DEB)),
+    locations = Locations(
+      accountName = "test",
+      regions = setOf(
+        ClusterRegion(
+          region = "us-east-1",
+          subnet = "internal (vpc0)",
+          availabilityZones = setOf("us-east-1c", "us-east-1d", "us-east-1e"),
+          image = VirtualMachineImage(
+            appVersion = appVersion,
+            id = usEastImageId
+          )
+        ),
+        ClusterRegion(
+          region = "us-west-2",
+          subnet = "internal (vpc0)",
+          availabilityZones = setOf("us-west-2a", "us-west-2b", "us-west-2c"),
+          image = VirtualMachineImage(
+            appVersion = appVersion,
+            id = usWestImageId
+          )
+        )
```

Comment:
When testing the code that converts `ClusterSpec` into `Map<String, ServerGroup>` we don't have to worry about cases for different `ImageProvider` types as whatever type is getting used, the AMI id would have been resolved by now.